thorlcr/activities/hashdistrib/thhashdistribslave.cpp

```diff
@@ -140,21 +140,46 @@ class CDistributorBase : implements IHashDistributor, implements IExceptionHandl
              */
             dedupList.sort(*iCompare, 1);
```

Comment:
@ghalliday @jakesmith Stable sort would be ideal but keeping things as they are doesn't makes things worse than non-best hash dedup. If multiple rows match dedup condition & BEST criteria, then the rows that retained may be different from the ones retained by Roxie or HThor.  However, this is no different from Hash Dedup without BEST where again if there are multiple rows with the same dedup condition, the retained rows may be different to Roxie and HThor.